packages/features/users/components/UserTable/InviteMemberModal.tsx

```diff
@@ -70,6 +71,7 @@ export function InviteMemberModal(props: Props) {
           role: values.role,
           usernameOrEmail: values.emailOrUsername,
           isPlatform: platformUser?.organization.isPlatform,
+          creationSource: CreationSource.WEBAPP,
```

Comment:
Would have liked to see this serverside only.

Reply:
Will try to address this in the follow-up. 🙏